vaadin-upload-flow-parent/vaadin-upload-flow/src/main/java/com/vaadin/flow/component/upload/Upload.java

```diff
@@ -649,6 +650,13 @@ private String getStringObject(String propertyName, String object,
         return result;
     }
 
+    /**
+     * Clear the fileList of the component.
+     */
+    public void clearFileList() {
+        getElement().setPropertyJson("files", Json.createArray());
```

Comment:
Use the `setFiles` method from the superclass (`GeneratedVaadinUpload`)

Reply:
Updated.